.gitignore

```diff
@@ -5,5 +5,6 @@ __pycache__/
 hc.sqlite
 hc/local_settings.py
 static-collected
+hc_venv/
```

Comment:
avoid adding virtual envs to your gitignore when working on team projects. Different people name their envs differently and if everyone added theirs the file would eventually become bloated with unnecessary lines. A better solution is to have the environment outside your project folder or better still use `virtualenv wrappers`